cub/cub/device/dispatch/dispatch_segmented_sort.cuh

```diff
@@ -557,7 +557,7 @@ template <typename LargeSegmentPolicyT,
           typename ValueT,
           typename BeginOffsetIteratorT,
           typename EndOffsetIteratorT>
-CUB_RUNTIME_FUNCTION cudaError_t
+CUB_RUNTIME_FUNCTION _CCCL_ATTRIBUTE_HIDDEN cudaError_t
```

Comment:
question: Wouldn't this need the `_CCCL_FORCEINLINE` to avoid potential intra-lib cross-TU crossover?

Reply:
No, there's no issue within a library, only across libraries